adapta/storage/distributed_object_store/v3/datastax_astra/_model_mappers.py

```diff
@@ -275,6 +277,143 @@ def _get_original_types(
         return {field.name: field.type for field in fields(self._data_model) if not subset or field.name in subset}
 
 
+class PanderaPolarsMapper(CassandraModelMapper):
```

Comment:
forgot to mentions this - let's forbid inheritance from top-level classes with `@final`

Reply:
Added